requirements/runtime.txt

```diff
@@ -1,3 +1,5 @@
+chumpy
+dataclasses
```

Comment:
is this pkg used?

Reply:
`dataclasses` is used by smplx package, if we remove it, the test will fail.
`chumpy` is used when we use the official SMPL weight file. When using the generated test weight file, it is not used. 
If we preprocess the official SMPL weight file, we can remove the chumpy element in that file. But the preprocess function also need to use chumpy package.

Reply:
* when you install `smplx`, so it will not install `dataclass`? does this mean that it is an issue with `smplx` package?
* `chumpy is used when we use the official SMPL weight file.` do we have code that do this? i didn't find any `import chumpy`

Reply:
- Because of the license of SMPL model, we can not distribute the official SMPL weight file. So in the test code, I generate a random weight file to initialize SMPL model, and the random weight file does not include chumpy elements. But in the training process, we need to use the official SMPL weight, which includes some elements need to be processed by chumpy package. Maybe I can add the dependency on chumpy in later PR.

- Under some setting, the dataclasses package is installed together with smplx,. But under some other setting, this package is not installed. I don't know the reason. Is there any possible solution?

Reply:
Yes add `chumpy` when it is actually used in codes.

Sounds like an upstream issue. Here is a possible solution:
we do not deal with dataclasses directly, and redirect all related error reports to the smplx repo.
we add an faq item somewhere in the docs, to help user diagnose&solve the problem related to missing dataclasses

btw, according https://github.com/ericvsmith/dataclasses , dataclasses supports >= python3.6, and will be included in python 3.7.
On the one hand, i'm not sure whether it works on python 3.5. On the other hand, it will be included in a newer python anyway.

Reply:
As you see, if I remove the dataclasses in the requirements, then some checks will fail. What should I do to solve this problem?